tasks/publish-index-image/tests/test-publish-index-image.yaml

```diff
@@ -58,14 +58,14 @@ spec:
               #!/usr/bin/env sh
               set -eux
 
-              internalRequests="$(kubectl get internalrequest --sort-by=.metadata.creationTimestamp --no-headers \
-              | tr "\n" " ")"
+              internalRequests=$(kubectl get internalrequest --sort-by=.metadata.creationTimestamp --no-headers \
+              -o json | jq '.items')
 
-              read -r -a ir <<< "${internalRequests}"
-              for((i=0; i<${#ir[@]}; i++)); do
+              irIndex=$(jq '. | length' <<< "${internalRequests}")
```

Comment:
Can we also check that there are exactly 2 of them and fail otherwise?

Reply:
check added.